gulpfile.js

```diff
@@ -74,6 +74,28 @@ gulp.task('umd', function(cb) {
       webpackCallBack('webpack', cb));
 });
 
+gulp.task('npm', function() {
+  var pkgJson = require('./package.json');
+  var targetPkgJson = {};
+  var fieldsToCopy = ['version', 'description', 'keywords', 'author', 'repository', 'license', 'bugs', 'homepage'];
+
+  targetPkgJson['name'] = '@ng-bootstrap/ng-bootstrap';
```

Comment:
It should, see: https://docs.npmjs.com/getting-started/scoped-packages#initializing-a-scoped-package

We just need to remember to publish it with `npm publish --access=public`